deploy/examples/csi/nfs/storageclass.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-nfs
+# Change "rook-ceph" provisioner prefix to match the operator namespace if needed
+provisioner: rook-ceph.nfs.csi.ceph.com # driver:namespace:operator
+parameters:
+  # nfsCluster is the name of the NFS-cluster as managed by Ceph.
+  nfsCluster: my-nfs
+
+  # server is the  Hostname, ip-address or service that points to the Ceph managed
```

The example just uses the service name, right? Should this always work? Maybe we don't need to document the host name or ip-address.
```suggestion
  # server is the host name, ip-address or service that points to the Ceph managed
```